demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -1317,6 +1330,7 @@ export class DemoMeetingApp implements AudioVideoObserver, DeviceChangeObserver,
 
   updateTilePlacement(tileIndex: number, x: number, y: number, w: number, h: number): void {
     const tile = document.getElementById(`tile-${tileIndex}`) as HTMLDivElement;
+    tile.classList.add(this.isContentTile(tileIndex) ? 'content-share-tile' : 'video-tile');
```

Comment:
When the tile element changes from the content share to the normal video, does the demo app replace this class with a new one?

Reply:
Good catch. I have updated the code to remove the previous class before adding a new one.